Makefile.am

```diff
@@ -327,9 +328,12 @@ COMMITTED_DIST = \
 	test \
 	$(NULL)
 
+node_modules/README: tools/README.node_modules
+	$(COPY_RULE)
```

Comment:
Oh, yes please. This should hopefully also get rid of that weird behaviour that more often than not, `node_modules/README` is missing after `npm install`. We constantly have to revert that before committing, and sometimes it even slips into PRs. So whatever the fate of the other commits is, please do land this one. :+1: